ChatQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -344,6 +321,8 @@ docker compose -f compose_vllm.yaml up -d
 
 ### Validate Microservices
 
+Note, before verify the microservices by curl or API, please make sure the **ports** of the microservices are opened in the firewall of the cloud node.
```

Comment:
Note, before verify the microservices by curl or API, please make sure the **ports** of the microservices are opened in the firewall of the cloud node if access from remote client.